cookbooks/aws-parallelcluster-platform/resources/enroot/partial/_enroot_common.rb

```diff
@@ -18,50 +18,36 @@
 action :setup do
   return if on_docker?
   action_install_package
-end
-
-action :configure do
-  return if on_docker?
-  return unless enroot_installed
 
-  cookbook_file "/tmp/enroot.template.conf" do
-    source 'enroot/enroot.template.conf'
-    cookbook 'aws-parallelcluster-platform'
+  template "/etc/enroot/enroot.conf" do
```

Comment:
what if the user already has their enroot configuration on their base AMI. Are we going to override their settings?

Reply:
It should not. Adding `action :create_if_missing` would solve the problem